src/main/external-resources/renderers/SkyBoxVRPlayer.conf

```diff
@@ -10,12 +10,13 @@ RendererIcon = SkyBoxVRPlayer.png
 # This renderer has sent the following strings:
 #
 # User-Agent: Linux/4.19.81+, UPnP/1.0, SKYBOX STUDIO for UPnP devices/1.14.12
+# User-Agent: Linux/4.19.81+, UPnP/1.0, Portable SDK for UPnP devices/17.1.5
 # ============================================================================
 
-UserAgentSearch = SKYBOX\s{1}STUDIO
+UserAgentSearch = Linux/4.19.81+
```

Comment:
It's a shame this is so generic. Can you add a test for it to this file to help us detect conflicts? https://github.com/UniversalMediaServer/UniversalMediaServer/blob/main/src/test/java/net/pms/configuration/RendererConfigurationTest.java

Reply:
I've added tests for SkyBox, Pigasus and Bigscreen VR Players.